index.html

```diff
@@ -295,26 +295,55 @@ <h1 class="greetings2">About Me</h1>
     </ul>
   </section>
 
-  <!-- Get In Touch-->
+  
   <section class="getintouch">
     <div class="sup-txt3">
       <p class="description5">
         I'm always interested in hearing about new projects, so if you'd like to chat please get in touch with me.
       </p>
     </div>
-    <form  method="post" class="get-in-touch-form">
-      <input class="common" type="text" placeholder="Full name" maxlength="30" name="fullname" required class="Name"/>
-        <!--Creating A line divider-->
-        <div class="line"></div>
-      <input class="common" type="email" placeholder="Email address" name="email" id="user-email" required />
-                <!--Creating A line divider-->
-        <div class="line"></div>
-      <textarea placeholder="Enter text here" id="" class="textArea" maxlength="500" name="comment"
-        required></textarea>
-      <button type="submit" class="btn-get">Get in touch</button>
+    <!-- Get In Touch Form STARTs here-->
+    <form  id="get-in-touch" action="https://formspree.io/f/mnqydzlw" method="post">
+      <!--Define the Full Name input-->
+      <ul class = "ul-form">
+        <li>
+          <input class="common" type="text" 
+                                placeholder="Full name" 
+                                maxlength="30"
+                                name="fullname" 
+                                required 
+                                class="Name"/>
+        </li>
+          <!--Creating A line divider-->
+          <div class="line"></div>
+          <!--Define the email input-->
+        <li>
+          <input class="common" type="email"
+                                placeholder="Email address"
+                                name="email" 
+                                required
+                                id="user-email" 
+                                />
+        </li>
+          <!--Creating A line divider-->
+          <div class="line"></div>
+          <!--Define Input Text Area-->
+        <li>
+          <textarea class ="textarea" 
+                    placeholder="Enter text here"
+                    name="user-message"
+                    maxlength="500"
+                     required></textarea>
+        </li>
+        <li class="button">
+          <button type="submit"
```

Comment:
- [ ] Kindly use the same button effects as mentioned in the Figma Design check below :point_down: 
![Screenshot from 2023-02-21 23-13-53](https://user-images.githubusercontent.com/45226848/220427534-b8c63973-2220-451d-afac-fd16a4aa981d.png)

Reply:
Thank you for the observation.

Reply:
Kindly use the same effect as requested above in your project  you are using the hover effect design for press event